lab-brianna/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lab-brianna",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha",
+    "start": "node index.js",
+    "lint": "eslint ./lib",
+    "build": "npm run lint && npm run test",
+    "install": "npm install"
```

Comment:
**TODO FOR LOST POINTS**

Take out Line 14. When I type ``npm install`` in my terminal, I get this endless loop of it trying to install. No need for an automated script for this command since we would type ``npm install`` or even better for shorthande, ``npm i`` to accomplish the same thing with installing dependencies.